tests/unit/test_main.py

```diff
@@ -121,7 +121,10 @@ def patch_bugtool(bugtool, mocker, dom0_template, report_name, tmp_path):
     def input(prompt):
         return "n" if "/proc" in prompt else "y"
 
-    bugtool.input = input
+    if sys.version_info.major == 2:  # pragma: no cover
+        bugtool.raw_input = input
```

Comment:
I don't understand the change. The code must use raw_input in python2, instead we're using the name bugtool.raw_input to actually use input.

I also don't understand why a conditional is needed in two codesites instead of just here, using `bugtool.input = raw_input`

Reply:
Oh, I see! With this I am overriding input (locally) and then I replace bugtool.raw_input with it.
```py
def input(prompt):
    return "n" if "/proc" in prompt else "y"
bugtool.raw_input = input
```
The test worked fine, but I can update it to use this instead to be more clear:
```py
def custom_input_simulator(prompt):
    """This custo input simulator is needed because we need to say "n" to files in /proc"""
    return "n" if "/proc" in prompt else "y"

bugtool.raw_input = custom_input_simulator
```
I hope this makes it more clear.